cmake/Modules/Platform/Emscripten.cmake

```diff
@@ -34,22 +34,10 @@ set(CMAKE_SYSTEM_PROCESSOR ${EMSCRIPTEN_SYSTEM_PROCESSOR})
 # This feature is activated if a shared library project has the property
 # SOVERSION defined.
 set(CMAKE_SHARED_LIBRARY_SONAME_C_FLAG "-Wl,-soname,")
+set(CMAKE_DL_LIBS "")
```

Comment:
This is setting the variable to an empty string, whereas the previous code was unsetting the variable. Does that matter?

Reply:
This is what other plaforms seem to do upstream.  They seem to either use the empty string or the string "dl"